raiden/transfer/log.py

```diff
@@ -83,6 +84,12 @@ def __init__(self, database_path):
             'FOREIGN KEY(statechange_id) REFERENCES state_changes(id)'
             ')'
         )
+        cursor.execute(
+            'CREATE TABLE IF NOT EXISTS state_events ('
+            'identifier integer primary key, source_statechange_id integer NOT NULL, data binary, '
+            'FOREIGN KEY(source_statechange_id) REFERENCES state_changes(id)'
+            ')'
+        )
         self.conn.commit()
 
     def write_transaction(self, data):
```

Comment:
Additionally, the signature in the class `StateChangeLogSQLiteBackend` is not matching the signature of the abc class `StateChangeLogStorageBackend`, I recommend the use of pyflakes+pylinter static checkers to catch these errors.